bfdd/bfd_packet.c

```diff
@@ -360,6 +367,16 @@ static int ptm_bfd_process_echo_pkt(struct bfd_vrf_global *bvrf, int s)
 		return -1;
 	}
 
+	/* RTT Calculation: add current RTT to samples */
+	if (my_rtt != 0) {
+		bfd->rtt[bfd->rtt_index] = my_rtt;
+		bfd->rtt_index++;
```

Comment:
there should be a warning, because my_rtt is int64_t and bfd->rtt[] is an uint64_t.
I would opt for uint64_t.

Reply:
changed all to uint64_t